### ISSUE TITLE
Updates to the L1 sticky behavior 

### ISSUE THREAD

**Issue by larahanlon2** [carbon-design-system/carbon-for-ibm-dotcom#714]
Referencing PR ["shrink masthead L1 when it becomes sticky".](https://github.com/carbon-design-system/ibm-dotcom-library/pull/674)
Here's my feedback:

1. There is a missing 16px padding or margin-right when the L1 shrinks. The title and blue underline shift position but this shouldn't happen. It's causing the entire L1 labels to move position. 

default:
<img width="1380" alt="Screen Shot 2019-11-20 at 1 05 50 PM" src="https://user-images.githubusercontent.com/53093550/69268899-f4059900-0b9d-11ea-9e45-736c46529839.png">

currently on shrink:
<img width="1381" alt="Screen Shot 2019-11-20 at 1 05 42 PM" src="https://user-images.githubusercontent.com/53093550/69268920-fec02e00-0b9d-11ea-9194-d42cad913fb5.png">

2. The size of the title chip (dark zone) is different on default and shrink. They should be the same. However, the width of the chip on default is too wide so that needs to be fixed. I'll open a new story for that.

3. We need to work out the behavior of L0 and L1 reappearing to default when user scrolls back up or wants to interact with masthead again. We talked about defaulting back when the user scrolls up for XXX amount of time OR moves their mouse to the top 25% of the screen. But there could be other ways of doing this.

4. On default, when opening an L1 dropdown the dropdown menu disappears on scroll. When the L1 is at 48px/shrunk and an L1 dropdown is open the dropdown menu stays open on scroll. It needs to be one or the other. My vote is that the dropdown should disappear on scroll for both. 

5. Adding to Wonil's comment about the need for animation or easing of the L1 when shrinking.

#### Additional Information
 - Here is a motion demo: https://ibm.box.com/s/cr7rqohnsidgonf77zvgwx1ye36ox6pr
 - These changes must be done before L1 hits the glass.  This severity will be adjusted based on the progress of the L1 issues.

**Comment by larahanlon2** [comment]
FYI @jeffchew @emerson-pereira @kennylam @annawen1 @wonilsuhibm @oliviaflory

**Comment by wonilsuhibm** [comment]
Thank you, Lara. 

0. Here is a motion demo: https://ibm.box.com/s/cr7rqohnsidgonf77zvgwx1ye36ox6pr

1 & 2. To add to these two items, the term `shrink` is misleading. As visible in the above demo, the L1 simply scrolls up with the rest of the page except it stops scrolling at some point, only leaving the bottom 48px of it visible and sticky.

3. To me there are two rules for reappearance.
- Rule 1: the user scrolls back up for X amount of pixels. Using time might be difficult.
- Rule 2: the scroll position is within X pixels from the top of the page.

4. Agreed. It should close. We may need a buffer here though to be able to compensate for scroll by accident. A dropdown closes only when the scroll delta is greater than X px.

5. Please read the above comment `1 & 2`. It's not an animation.

**Comment by RobertaJHahn** [comment]
Nov 20, 2019 Slack comment from Scott:  the L0 & L1 will be informed by the self service menu for drupal pages. as for other backends used to create L1 nav, we’d have to talk to those teams.  Ishita is the lead and would know best around timing of the self service menus.

Nov 20, 2019 Slack comment from Jeff:  we have been talking about what the data structure should look like for the L1 in order to populate the nav, we probably should round back to Ishita to discuss

**Comment by RobertaJHahn** [comment]
We will look at this issue when dev hooks up the self service Nav with L1.  Moving to icebox.

**Comment by RobertaJHahn** [comment]
This issue has been placed in the icebox pipeline to be reviewed in the future.  Keeping it open.

**Comment by larahanlon2** [comment]
@RobertaJHahn we will also need a new issue for updating the L1 design. We made slight design modifications to parts of the L1 in Q4 2019 which will need a dev story when revisiting this feature. I will open an issue and move to icebox.

**Comment by RobertaJHahn** [comment]
Keep for historical purposes. There are more design explorations going on.
